Validation/TrackerRecHits/src/SiStripRecHitsValid.cc

```diff
@@ -594,26 +582,30 @@ void SiStripRecHitsValid::createMEs(DQMStore::IBooker & ibooker,const edm::Event
     std::string label = hidmanager.getSubdetid(detid,tTopo,true);
     // std::cout << "label " << label << endl;
       
-    std::map<std::string, LayerMEs>::iterator iLayerME  = LayerMEsMap.find(label);
```

Comment:
do not understand why getSubdetid returns a string and not a enumerator
if possible migrate all this code to use enumerators